templates/zerver/app/markdown_help.html

```diff
@@ -128,6 +128,22 @@
 ```</td>
                     <td class="rendered_markdown"><blockquote><p>Quoted block</p></blockquote></td>
                     </tr>
+                    <tr>
+                        <td class="preserve_spaces">```spoiler Spoiler: Cars (the text after spoiler is an optional header)
+Car spoilers reduce drag on a car. This text won't be visible until the user clicks.
+```</td>
+                        <td class="rendered_markdown">
+                            <div class="spoiler-block">
+                                <div class="spoiler-header"><a class="spoiler-button"><span class="spoiler-arrow spoiler-button-open"></span></a>
+                                    <p>Spoiler: Cars (the text after spoiler is an optional header)</p>
+                                </div>
+
+                                <div class="spoiler-content spoiler-content-open">
+                                    <p>Car spoilers reduce drag on a car. This text won't be visible until the user clicks.</p>
+                                </div>
+                            </div>
+                        </td>
+                    </tr>
```

Comment:
Changed the example to be more focused on explanation (no mention of cars).

We should probably do a follow-up project to avoid copying rendered markdown into a static HTML file.  In an idea world, we'd just have a bit of JavaScript with a handlebars template to just render the examples using the markdown processor; I think all of the examples here should agree with the backend rendering and let us delete all the generated code here.  Opened this as https://github.com/zulip/zulip/issues/15375.